tf-modules/archive/bootstrap.tf

```diff
@@ -1,12 +1,4 @@
-resource "null_resource" "rsa_keys" {
-  triggers = { x = uuid() }
-  provisioner "local-exec" {
-    command = "${path.module}/fetch_or_create_rsa_keys.sh ${var.system_bucket} ${var.prefix}"
```

Comment:
Does that mean this `fetch_or_create_rsa_keys.sh` script can be deleted as well?

Reply:
Good catch, I thought I had.